zipline/data/us_equity_pricing.py

```diff
@@ -1316,3 +1329,33 @@ def get_stock_dividends_with_ex_date(self, assets, date, asset_finder):
         c.close()
 
         return stock_divs
+
+    def unpack_db_to_component_dfs(self):
+        """Returns the set of known tables in the adjustments file in DataFrame
+        form.
+
+        Returns
+        -------
+        dfs : dict{str->DataFrame}
+            Dictionary which maps table name to the corresponding DataFrame
+            version of the table, where all date columns have been coerced back
+            from int to datetime.
+        """
+
+        def _get_df_from_table(table_name, date_cols):
+
+            # Dates are stored in second resolution as ints in adj.db tables.
+            return read_sql(
+                'select * from "{}"'.format(table_name),
+                self.conn,
+                index_col='index',
+                parse_dates={col: 's' for col in date_cols}
+            ).rename_axis(None)
```

Comment:
Ah interesting (seems like we might not even need to be writing the index column in the first place, but not worth worrying about here)